.github/workflows/ci.yml

```diff
@@ -108,6 +110,43 @@ jobs:
           cache-from: type=registry,ref=${{ env.DOCKER_IMAGE_TAG }}
           cache-to: type=inline
 
+  tag-publish:
+    if: (github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/'))
+    needs: [ prettier, es-lint, tests-finish ]
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_CACHE_SOURCE: staging
+      IMAGE_REGISTRY: safeglobal
+      IMAGE_NAME: safe-client-gateway-nest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Extract commit metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
```

Comment:
Do we need this action?

Reply:
> If we restrict the target to releases I think we could just use `github.event.release.tag_name` and remove this action 👍

You don't need the release event or this action to get the tag name. You can get it via `github.ref_name`. https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

This PR should accomplish the same thing (but for the staging action): https://github.com/safe-global/safe-client-gateway-nest/pull/517

Reply:
Changed in [494a482](https://github.com/safe-global/safe-client-gateway-nest/pull/525/commits/494a482d3dfb77fdcf3eed5ca9ab5a3e4b7ba808)